frontend/src/types/AnnotationUIContext.ts

```diff
@@ -240,6 +240,7 @@ export class AudioAnnotationUIContext {
   constructor() {
     this.container_width = 0
     this.container_height = 0
+    this.has_changed = false
```

Comment:
Had to add this so that reactive dependency is created for `AudioAnnotationUIContext` instance `has_changed` property.